iFixit/Bootstrap-Form-Builder
# Save Button: disable while saving

This will hopefully fix [an issue](https://sentry.io/organizations/ifixit/issues/2550111710/?project=5514638&referrer=github_integration) where customers were, seemingingly, double-submitting this Data Capture Step form.

QA
---
When editing a Data Capture form on Step Edit, the "Save" button should now be disabled while the form is submitting.

Also, if the form submit fails, I added a "failed" message.

Fixes PHP-87W
Closes https://github.com/ifixit/ifixit/issues/39580
